Security.md

```diff
@@ -1,28 +0,0 @@
-# Security
```

Comment:
this file was a combo of personal to-do list and points that can otherwise be issues.